modules/veteran_verification/app/services/bgs/mock_disability_rating_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'savon'
+require 'nokogiri'
+
+module BGS
```

Comment:
I'm not seeing a spec for this service class? Is there a reason for not adding one?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/ad4fe80d65cf3eb50d00d9a5e4addc2c3e3f847f/modules/veteran_verification/spec/requests/mock_disability_rating_request_spec.rb

it's located here, next to the non-mocked service test.